director/tests/test_urls.py

```diff
@@ -49,7 +49,7 @@ def check(path, anon=None, joe=None, mary=None):
 # Skip a test
 # Useful to quickly skipping a test
 # instead of having to comment out multiple lines
-def Skip(path, *args, **kwargs):
+def skip(path, *args, **kwargs):
```

Comment:
https://docs.python.org/3/library/unittest.html#unittest.skip

Reply:
This is different to `unittest.skip`, it's skipping a "check" within a unit test. Updated comment to reflect that.